### PR TITLE
Add Public USSD line flow

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/ussd_public.js [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+                    new Choice('yes_hiv_msgs', $('Yes')),
+                    new Choice('no_hiv_msgs', $('No'))
+                ],
+                next: 'state_end_thank_you_enter'
+            });
+        });
+
+        // Interstitial
+        self.add('state_end_thank_you_enter', function(name) {
+            return self.im.outbound.send_to_user({
```
Comment (imsickofmaps) [inline src/ussd_public.js]: Remove this - we're going to send confirmations from Reg service now

File: test/ussd_public.test.js [hunk 2]
```diff
@@ -0,0 +1,584 @@
...
+                        assert.equal(sms.content,
+                            "Welcome to FamilyConnect. 's FamilyConnect ID is 7777.  Write it down and give it to the Nurse at your next clinic visit."
+                        );
+                        assert.equal(sms.to_addr,'082222');
+                    })
+                    .run();
+            });
+        });
+
+        describe.only("Change testing", function() {
```
Comment (imsickofmaps) [inline test/ussd_public.test.js]: o_O

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**